requirements-dev-minimal.txt

```diff
@@ -1,6 +1,7 @@
 astroid
 asynctest
 coverage
+debugpy
```

Comment:
```suggestion
# VSCode debugging requirement
# See https://code.visualstudio.com/docs/python/debugging#_local-script-debugging
debugpy
```

might be helpful since there's no other reference to this library in this repo so an enterprising yelper trying to cleanup the unused dependencies for this repo might remove this unknowingly :)